scipy-1.0/paper.tex

```diff
@@ -693,11 +693,11 @@ \subsection*{Project scope}
 PyMC3\cite{10.7717/peerj-cs.55}, emcee\cite{2013PASP-emcee} and 
 PyStan\cite{pystan-ref} cover Bayesian statistics and probabilistic modeling.
 scikit-image\cite{vanderwalt2014scikit} provides image processing
-capabilities beyond \texttt{scipy.ndimage}, sympy\cite{meurer2017sympy}
```

Comment:
the notation `scipy.x` is not introduced, while my proposed change here is more consistent with the reference to `sparse` and `spatial` later in the sentence

Reply:
Changing to SciPy's \texttt{ndimage} makes sense if we do not expect the reader to understand the subpackage notation. I'll leave that to others to decide.